community/server/src/test/java/org/neo4j/server/rest/transactional/TransactionHandleTest.java

```diff
@@ -97,9 +99,9 @@ public void shouldExecuteStatements() throws Exception
         InOrder outputOrder = inOrder( output );
         outputOrder.verify( output ).transactionCommitUri( uriScheme.txCommitUri( 1337 ) );
         outputOrder.verify( output ).statementResult( executionResult, false, (ResultDataContent[])null );
-        outputOrder.verify( output ).notifications( anyCollectionOf( Notification.class ) );
+        outputOrder.verify( output ).notifications( anyCollection() );
         outputOrder.verify( output ).transactionStatus( anyLong() );
-        outputOrder.verify( output ).errors( argThat( hasNoErrors() ) );
+        outputOrder.verify( output ).errors( MockitoHamcrest.argThat( hasNoErrors() ) );
```

Comment:
You didn't want a static import of `argThat` because it's not the familiar Hamcrest function?